beacon-chain/blockchain/service.go

```diff
@@ -147,10 +147,12 @@ func (c *ChainService) updateHead(processedBlock <-chan *types.Block) {
 				continue
 			}
 
-			cState := c.beaconDB.GetCrystallizedState()
-			aState := c.beaconDB.GetActiveState()
-			var stateTransitioned bool
+			aState, cState, err := c.beaconDB.GetState()
```

Comment:
get state can sound a bit ambiguous - am i retrieving the "canonical" state or is it a generic function to retrieve any state pair from the db even from previous blocks?

Reply:
good point. i'm going to remove this, as I think it's going to be deprecated soon anyways.